tests/cnn-visualizer.test.js

```
@@ -0,0 +1,208 @@
+          if (this.onload) this.onload();
+        }, 0);
+      }
+    };
+
+    global.Event = class {
+      constructor(type) {
+        this.type = type;
+      }
+    };
+
+    eval(cnnCode);
```

Biome (2.5.3)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)